pgxn/neon/pagestore_smgr.c

```diff
@@ -2843,7 +2843,7 @@ neon_slru_read_page(SlruCtl ctl, int segno, BlockNumber blkno, XLogRecPtr lsn, c
 	NeonResponse 				*resp;
 	NeonGetSlruPageResponse 	*get_slru_page_resp;
 	NeonSlruKind	kind;
-	bool			latest;
+	bool			latest = false;
```

Comment:
If the second `if` statement in this function is false then this `latest` variable will be used without being initialized, which could be problematic.